README.md

```diff
@@ -1,14 +1,16 @@
-# todo-list
+# Todo-list
 
-> A Vue.js project
+> A Vue.js project.
+## Description
+Todo-list allows a user to add tasks. There is a window showing all tasks. The newly added tasks will be shown at top of the task window. User can check or uncheck each task to mark if it is completed and hide/show completed tasks by clicking hide/show button. Try at: http://todo.senhung.net/
 
 ## Build Setup
```

Comment:
Can you help me to remove the entire Build Setup block? It comes with the Vue webpack project initialization.